app/mailers/project_summary_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectSummaryMailer < ApplicationMailer
+
+  def project_summary(user)
+    @user = user
+    mail(to: user.email, subject: "[Learn IT, Girl] Daily summary of your board", sent: DateTime.now)
```

Comment:
sorry read you comment :P anyway if you want to put date in subject please do it like that "[Learn IT, Girl] Daily summary of your board - #{DateTime.now.strftime("%d %b %Y")}".